src/dms-preview/azext_dms/custom.py

```diff
@@ -54,8 +55,9 @@ def create_or_update_project(
         # because of this we need to raise the error here.
         try:
             # TODO: Remove this check after validations are added to core
-            if scenario_handled_in_extension:
-                raise ValueError
+            if source_platform != "sql" or target_platform != "sqldb":
```

Comment:
I think this should be a method as it's used in more than one place. If possible, maybe extension_handles_scenario could be modified?

Reply:
The method scenario_handled_in_extension should only filter out scenarios that are in the extension. The validation here should only validate that the only scenario we're pushing to the core method is only the sql->sqldb scenario. This way, if a user puts in a scenario that isn't overridden in the extension, we don't send any non supported scenarios to the core. For example, say the user selected sql -> mongosql: since this is not a scenario supported by the extension, this will be sent to the core. 

Ideally, this should be in the scenario handled in the core. When we add this method to the core, we can remove this line completely from the extension.

Reply:
I see what you're saying. We should probably get this check into the core as it confuses me every time I look at it with fresh eyes :)

Reply:
Agreed :)